mesonbuild/scripts/python_info.py

```diff
@@ -63,10 +62,15 @@ def get_install_paths():
     return paths, install_paths
 
 def links_against_libpython():
-    from distutils.core import Distribution, Extension
-    cmd = Distribution().get_command_obj('build_ext')
-    cmd.ensure_finalized()
-    return bool(cmd.get_libraries(Extension('dummy', [])))
+    # on versions supporting python-embed.pc, this is the non-embed lib
+    if sys.version_info >= (3, 12):
```

Comment:
I've changed this from `(3, 8)` (I think) to `(3, 12)`. I've already explained the reason.